### PR TITLE
Add Startup with IBM Program Info

### PR DESCRIPTION
**Pull request by bwentwor**
A new section with added called "How do I get answers to my Startup with IBM questions?" This section is needed to address numerous chat assistant and support questions.

### CODE REVIEW

File: billing-faq.md [hunk 1]
```diff
@@ -136,6 +136,12 @@ Credit might take a few hours to appear in your account. To see whether a credit
 
 If the credit isn't on the Usage page, go to **Invoices** and click link with the date for your next recurring invoice. If you don't see the credit on the next recurring invoice, it’s not yet added to your account. Check back later to verify that you received the credit.
 
+## How do I get answers to my Startup with IBM questions?
+{: #startup-ibm}
+{: faq}
+
+Information about the Startup with IBM Program, which was formerly the IBM Global Entrepreneur Program (GEP), is available at [Startup with IBM Program](https://developer.ibm.com/startups/){: external} web site. The awarding and extension of credits through this IBM corporate program is not directly supported by IBM Cloud Support. If your application to the program is approved, credits might be referred to as the Technology Incubator Program on an IBM Cloud Invoice. Answers to questions about the program and its credits are available in the [Frequently Asked Questions for the Startup with IBM Program](https://developer.ibm.com/startups/faq){: external}.
```
Comment (hooznora) [inline billing-faq.md:143]: - Regarding the first sentence, I would change "at" to "**on the** [Startup with IBM Program](https://developer.ibm.com/startups/){: external} **website**.

- Suggestion for the last sentence: For more information about the program and its credits, visit the [Frequently Asked Questions for the Startup with IBM Program page](https://developer.ibm.com/startups/faq){: external}, where you can find helpful answers to a number of questions from different topics.

Comment (bwentwor) [reply billing-faq.md:143]: @hooznora I agree with your suggestions. :-) Can you please make them and commit them?

**Comment by bwentwor** [comment]
@hooznora The information is now live at the following URL:
https://cloud.ibm.com/docs/billing-usage?topic=billing-usage-billusagefaqs
Can you close this issue?

**Comment by hlucey** [comment]
Confirmed with @hooznora that this was implemented.

**Pull request closed without merging**